### ISSUE TITLE
Overflow when casting from double to time_t, and_PyTime_t.

### ISSUE THREAD

**Issue by c52bddf0-d054-40a8-9f85-1ccf099957bc** [python/cpython#78604]
BPO | [34423](https://bugs.python.org/issue34423)
--- | :---
Nosy | @malemburg, @mdickinson, @abalkin, @vstinner, @pganssle, @enedil
PRs | <li>python/cpython#8802</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2018-08-17.21:56:49.149>
labels = ['interpreter-core', '3.8', 'type-bug', '3.7']
title = 'Overflow when casting from double to time_t, and_PyTime_t.'
updated_at = <Date 2019-02-20.11:13:50.802>
user = 'https://github.com/enedil'
```

bugs.python.org fields:
```python
activity = <Date 2019-02-20.11:13:50.802>
actor = 'vstinner'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Interpreter Core']
creation = <Date 2018-08-17.21:56:49.149>
creator = 'enedil'
dependencies = []
files = []
hgrepos = []
issue_num = 34423
keywords = ['patch']
message_count = 5.0
messages = ['323676', '323929', '323930', '323959', '336072']
nosy_count = 6.0
nosy_names = ['lemburg', 'mark.dickinson', 'belopolsky', 'vstinner', 'p-ganssle', 'enedil']
pr_nums = ['8802']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue34423'
versions = ['Python 3.6', 'Python 3.7', 'Python 3.8']
```

</p></details>

**Comment by iritkatriel** [comment]
Closed along with the PR.

**Comment by mdickinson** [comment]
Fixed for Python 3.10 and later by #101826 and its backports.
